composer.json

```diff
@@ -10,13 +10,13 @@
   ],
   "minimum-stability": "dev",
   "require": {
-    "php": "^7.1.3",
-    "illuminate/notifications": "~5.8.0|^6.0|^7.0|^8.0",
+    "php": "^7.1.3|^8.0",
+    "illuminate/notifications": "~5.8.0|^6.0|^7.0|^8.0|^8.20",
```

Comment:
Wouldn't `^8.0` here work with 8.20 as well or do we need to support 8.20 as a minimum?